src/Integration/Resources/Strings.resx

```diff
@@ -433,4 +433,8 @@ Localization note: '[Don't warn me again]()' is a special markup used to transla
     <value>Not all NuGet packages were installed. Please see output above for more information.</value>
     <comment>Output message indicating not all NuGet packages were installed during the binding workflow</comment>
   </data>
+  <data name="SolutionSaveCancelledBindAborted" xml:space="preserve">
+    <value>Solution save was cancelled. Bind operation was aborted.</value>
```

Comment:
How about:
`User cancelled the solution save operation. Binding cannot continue with unsaved files. Binding was aborted.`
?